src/modules/numbers/numbers.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable, NotFoundException} from '@nestjs/common';
+import { CreateNumberDto } from './dto/create-number.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { UkefNumber as Number } from './entities/number.entity';
+
+@Injectable()
+export class NumbersService {
+
+  constructor(
+    @InjectRepository(Number, "mssql-number-generator")
+      private readonly numberRepository: Repository<Number>,
```

Comment:
can we change the `Number` to a primitive version (`number` - in lowercase)

Reply:
renamed to ukefId. Number was referring to output of number generator, that is ukef id object, and not primitive number.